tests/cypress/integration/actions_objects/case_60_autoborder_feature.js

```diff
@@ -67,11 +63,6 @@ context('Autoborder feature.', () => {
             });
     }
 
-    function testActivatingShape(x, y, expectedShape) {
-        cy.get('.cvat-canvas-container').trigger('mousemove', x, y);
-        cy.get(expectedShape).should('have.class', 'cvat_canvas_shape_activated');
-    }
```

Comment:
Why did you remove this check?

Reply:
Because it is not necessary anymore. Previous test implementation activated shapes to get coordinates of circles. 
These coordinates needed to check final result. The test worked this way, but it was a kind of lucky. Circles for different shapes are located in different coordinate spaces and comparing them is incorrect way. Two points in different places on canvas can have the same coordinates (0,0 for example). I've rewritten the test and now it gets correct position from shape attributes.